modules/tinymce/src/plugins/pagebreak/test/ts/browser/PageBreakSplitBlockTest.ts

```diff
@@ -101,7 +101,7 @@ describe('browser.tinymce.plugins.pagebreak.PageBreakSplitBlockTest', () => {
       clickPageBreak(editor);
 
       const content = editor.getContent({ source_view: true });
-      assert.equal(content, `<p>some<!-- pagebreak -->text</p>`);
+      assert.equal(content, `<p>some<div style="break-after: page"></div>text</p>`);
```

Comment:
Oh shit I think this might need to be a span or something that is not valid HTML.

Reply:
Or at least we need to make sure that `pagebreak_split_block` is enabled by default if it's a div.